app/src/features/apiClient/components/RouteElement.tsx

```
@@ -1,20 +1,194 @@
+          new Promise((res) => {
+            setTimeout(() => res(v), 1000 * 3);
+          })
+      );
+
+    registerWorkflow({
+      cancelWarning: "Request is still in progress. Close tab?",
+      workflow: {
+        abort: () => {
+          abortController.abort();
+        },
+        then: (callback: () => void) => {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)